server/server.go

```diff
@@ -193,6 +195,7 @@ func (s *Server) initHTTP() {
 	s.mux.Handle(kv.RESTPrefix, s.kvREST)
 	s.mux.Handle(kv.DBPrefix, s.kvDB)
 	s.mux.Handle(structured.StructuredKeyPrefix, s.structuredREST)
+	s.mux.Handle(ts.UrlPrefix, s.tsServer)
```

Comment:
Capitalized URL in these variables. I didn't rename the other prefixes in this change, but I do think we could use a consistency sweep through our server components.